.travis.yml

```diff
@@ -12,14 +12,18 @@ before_deploy: |
   if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
     cabal install -f 'static'
     tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    curl https://github.com/lindell/github-release-cli/releases/download/1.3.0/github-releaser-travis -L --output github-releaser && chmod +x github-releaser
+    export GITHUB_OAUTH_TOKEN="$GITHUB_TRAVIS_DEPLOY"
+    export RELEASE_NAME=Dotenv-Release-v$(cabal exec dotenv -- --version)
+    export BODY=$(envsubst < ./CHANGELOG.md)
```

Comment:
@CristhianMotoche I'mnot sure about adding the whole `CHANGELOG` to each release, it might too verbose, perhaps we only need to include the latest changes.

Reply:
You're right @sestrella Let me add some script to filter the change log.